fastlane/spec/plugin_manager_spec.rb

```diff
@@ -118,6 +118,27 @@
       end
     end
 
+    describe "Previously bundled action" do
+      it "#formerly_bundled_actions returns an array of string" do
+        expect(Fastlane::Actions.formerly_bundled_actions).to be_kind_of(Array)
+
+        Fastlane::Actions.formerly_bundled_actions.each do |current|
+          expect(current).to be_kind_of(String)
+        end
+      end
+
+      it "shows how to install a plugin if you want to use a previously bundled action" do
+        deprecated_action = "deprecated"
+        expect(Fastlane::Actions).to receive(:formerly_bundled_actions).and_return([deprecated_action])
+
+        expect do
+          result = Fastlane::FastFile.new.parse("lane :test do
+            #{deprecated_action}
+          end").runner.execute(:test)
+        end.to raise_exception("The action '#{deprecated_action}' is no longer bundled with fastlane. You can install it using `fastlane add_plugin deprecated`")
+      end
+    end
```

Comment:
Can we add a test to verify it doesn't show this message if the plugin is installed or do you think this case is covered @KrauseFx  ?